ArcGISRuntimeSDKQt_CppSamples/Scenes/OpenMobileScenePackage/README.md

```diff
@@ -0,0 +1,35 @@
+# Open mobile scene package
+
+Opens and displays a scene from a Mobile Scene Package (.mspk).
+
+![](screenshot.png)
+
+## Use Case
+An .mspk file is an archive containing the data (specifically, basemaps and features), used to display an offline 3D scene.
+
+## How it works
+1. Create a `MobileScenePackage` using the path to the local `.mspk` file.
+2. Use `MobileScenePackage::isDirectReadSupported` to check whether the package can be read in the archived form (.mspk) or whether it needs to be unpacked.
```

Comment:
these steps are not correct: currently you don't create the package until you have done the isDirectReadSupported check - but as discussed, maybe we can do load and handle errors instead

Reply:
ok. this is from the common design, so we probably need to get that updated in that case.